packages/main/src/themes/TextArea.css

```diff
@@ -166,158 +259,155 @@
 }
 
-:host([value-state="Success"]:not([readonly]):not([focused]):hover) .ui5-textarea-inner {
-	box-shadow: var(--sapContent_Positive_Shadow);
-	background-color: var(--sapField_Hover_Background);
+:host([value-state="Success"][focused]:not([readonly])) .ui5-textarea-wrapper {
+	background-color: var(--_ui5_textarea_focused_value_state_success_background);
+	border-color: var(--_ui5_textarea_focused_value_state_success_border_color);
 }
 
-:host([value-state="Success"][focused]:not([readonly])) .ui5-textarea-inner {
-	background-image: none;
-	box-shadow: var(--_ui5_textarea_value_state_success_focus_box_shadow);
-	background-color: var(--sapField_Focus_Background);
+:host([value-state="Success"][focused]:not([readonly])) .ui5-textarea-wrapper:after {
+	border-color: var(--_ui5_textarea_focused_value_state_success_focus_outline_color);
 }
 
-:host([value-state="Error"]:not([readonly])) {
-	border-color: var(--sapField_InvalidColor);
+:host([value-state="Success"]:not([readonly])) .ui5-textarea-wrapper::before{
+	background-color: var(--_ui5_textarea_value_state_success_border_botom_color);
 }
 
-:host([value-state="Error"]:not([readonly])) .ui5-textarea-inner {
-	background-color: var(--sapField_InvalidBackground);
+:host([value-state="Success"]:not([readonly]):not([focused]):hover) .ui5-textarea-wrapper {
+	background-color: var(--sapField_Hover_Background);
+	box-shadow: var(--_ui5_textarea_value_state_success_hover_box_shadow);
 }
 
-:host([value-state="Error"]:not([readonly]):not([disabled])) .ui5-textarea-inner,
-:host([value-state="Warning"]:not([readonly]):not([disabled])) .ui5-textarea-inner,
-:host([value-state="Information"]:not([readonly]):not([disabled])) .ui5-textarea-inner {
-	border-style: var(--_ui5_input_error_warning_border_style);
+:host([value-state][value-state="Success"]) .ui5-textarea-inner {
+	border-radius: var(--sapField_BorderCornerRadius);
 }
 
-:host([value-state="Warning"]:not([readonly])) {
-	border-color: var(--sapField_WarningColor);
+:host([value-state="Information"]) .ui5-textarea-inner{
+	padding-top: var(--_ui5_textarea_padding_top_information);
+	padding-bottom: var(--_ui5_textarea_padding_bottom_information);
 }
 
-:host([value-state="Warning"]:not([readonly])) .ui5-textarea-inner {
-	background-color: var(--sapField_WarningBackground);
+:host([value-state="Information"]:not([readonly]):not([disabled])) .ui5-textarea-wrapper {
+	background: var(--sapField_InformationBackgroundStyle);
+	background-color: var(--sapField_InformationBackground);
+	border-color: var(--_ui5_textarea_value_state_information_border_color);
 }
 
-:host([value-state="Success"]:not([readonly])) {
-	border-color: var(--sapField_SuccessColor);
+:host([value-state="Information"][focused]:not([readonly])) .ui5-textarea-wrapper {
+	background-color: var(--_ui5_textarea_focused_value_state_information_background);
+	border-color: var(--_ui5_textarea_focused_value_state_information_border_color);
```

Comment:
border-color: var(--_ui5_textarea_focused_value_state_information_border_color); the only value applied for all supported themes is var(--sapField_InformationColor); do we need a parameter